kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md

```diff
@@ -4,6 +4,9 @@ The sample scripts demonstrate the creation of a WebLogic domain home on an exis
 
 ## Prerequisites
 
+Please read this guide before proceeding.
+* [Domain Resource](../../../../site/domain-resource.md)
+
 The following prerequisites must be handled prior to running the create domain script:
 * Make sure the WebLogic Operator is running.
```

Comment:
Operator -> operator (globally; only capitalize the term when it is used in its full name, WebLogic Server Kubernetes Operator, otherwise use "operator.")